.circleci/config.yml

```diff
@@ -136,6 +136,22 @@ jobs:
     - run:
         name: "Static analysis"
         command: ./do qa:phpstan
+  qa_js:
+    executor: wpcli_php_oldest
```

Comment:
Why "oldest"?

Reply:
no reason, I copied that from the `js_tests` job